.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,91 @@
+# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Create a bug report
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: [requires triage]
+body:
+  - type: markdown
+    attributes:
+      value: "
+
+**FOR ALL USERS - PLEASE READ THIS PARAGRAPH AS IT HELPS US TO DO OUR WORK MORE EFFICIENTLY:**
```

```suggestion
**PLEASE READ THIS PARAGRAPH - this helps us identify the root cause and fix the issue:**
```

it feels like too many all-caps otherwise 😄